### ISSUE TITLE
Interoperability with Netapp handling 501 reply for request with versioning parameter

### ISSUE THREAD

**Issue by beisstor** [iterate-ch/cyberduck#13683]
This ticket system is to report bugs and feature requests. For support, visit the [help page](https://cyberduck.io/help/) first.

Connect S3 from a Mac Book to a Netapp ONTAP 9.9.1.

**Describe the bug**
Everthing works fine. After an update to cyberduck 8.4.3. I got am error. Look at the screen shoot.

**To Reproduce**
If I reinstall Version 8.4.2. Everthing works fine.

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/76955322/186078390-9112f9e3-d67c-4184-b92a-848116f9d0a5.png)

**Desktop (please complete the following information):**
 - OS: 12.5.1

**Log Files**
Please include the [log file](https://docs.cyberduck.io/cyberduck/support/#logging-output). You can reach the logging output in _Preferences → Connection_.

**Additional context**

**Comment by dkocher** [comment]
Please include the [log file](https://docs.cyberduck.io/cyberduck/support/#logging-output). You can reach the logging output in Preferences → Connection.

**Comment by beisstor** [comment]
Regards

Thorsten

**Comment by beisstor** [comment]
[cyberduck.log](https://github.com/iterate-ch/cyberduck/files/9400365/cyberduck.log)

**Comment by dkocher** [comment]
> [cyberduck.log](https://github.com/iterate-ch/cyberduck/files/9400365/cyberduck.log)

Can you please enable _Debug_ output in _Preferences → Connection → Log_.

**Comment by beisstor** [comment]
the test with debug.
regards
[cyberduck.log](https://github.com/iterate-ch/cyberduck/files/9402919/cyberduck.log)

**Comment by dkocher** [comment]
The null pointer to be fixed in #13686 is hiding the actual failure in this issue.

 * Trancsript
```
2022-08-23 15:18:59,912 [Thread-53] INFO  ch.cyberduck.transcript.request - GET /k8s-sentinel-one/?versioning HTTP/1.1
2022-08-23 15:18:59,912 [Thread-53] INFO  ch.cyberduck.transcript.request - Date: Tue, 23 Aug 2022 13:18:59 GMT
2022-08-23 15:18:59,912 [Thread-53] INFO  ch.cyberduck.transcript.request - x-amz-content-sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
2022-08-23 15:18:59,912 [Thread-53] INFO  ch.cyberduck.transcript.request - Host: nas58-svm-s3.freenet-group.de
2022-08-23 15:18:59,912 [Thread-53] INFO  ch.cyberduck.transcript.request - x-amz-date: 20220823T131859Z
2022-08-23 15:18:59,912 [Thread-53] INFO  ch.cyberduck.transcript.request - Authorization: ********
2022-08-23 15:18:59,912 [Thread-53] INFO  ch.cyberduck.transcript.request - Connection: Keep-Alive
2022-08-23 15:18:59,912 [Thread-53] INFO  ch.cyberduck.transcript.request - User-Agent: Cyberduck/8.4.3.38269 (Mac OS X/12.5.1) (x86_64)
2022-08-23 15:18:59,912 [Thread-53] INFO  ch.cyberduck.transcript.request - Accept-Encoding: gzip,deflate
2022-08-23 15:18:59,934 [Thread-53] INFO  ch.cyberduck.transcript.response - HTTP/1.1 501 Not Implemented
2022-08-23 15:18:59,934 [Thread-53] INFO  ch.cyberduck.transcript.response - Server: NetApp CSS/9.9.1P5
2022-08-23 15:18:59,934 [Thread-53] INFO  ch.cyberduck.transcript.response - Date: Tue, 23 Aug 2022 13:18:59 GMT
2022-08-23 15:18:59,934 [Thread-53] INFO  ch.cyberduck.transcript.response - Connection: Keep-Alive
2022-08-23 15:18:59,934 [Thread-53] INFO  ch.cyberduck.transcript.response - Content-Length: 167
2022-08-23 15:18:59,934 [Thread-53] INFO  ch.cyberduck.transcript.response - Content-Type: application/xml
2022-08-23 15:18:59,934 [Thread-53] DEBUG org.apache.http.impl.execchain.MainClientExec - Connection can be kept alive indefinitely
```

 * Log
```
2022-08-23 15:18:59,934 [Thread-53] WARN  ch.cyberduck.core.http.DisabledServiceUnavailableRetryStrategy - No retry for response HttpResponseProxy{HTTP/1.1 501 Not Implemented [Server: NetApp CSS/9.9.1P5, Date: Tue, 23 Aug 2022 13:18:59 GMT, Connection: Keep-Alive, Content-Length: 167, Content-Type: application/xml] ResponseEntityProxy{[Content-Type: application/xml,Content-Length: 167,Chunked: false]}}
2022-08-23 15:18:59,935 [Thread-53] DEBUG org.jets3t.service.impl.rest.httpclient.RestStorageService - Rethrowing as a ServiceException error in performRequest: org.jets3t.service.ServiceException: Service Error Message. -- ResponseCode: 501, ResponseStatus: null, XML Error Message: <?xml version="1.0" encoding="UTF-8"?><Error><Code>NotImplemented</Code><Message>A header you provided implies functionality that is not implemented.</Message></Error>

```

**Comment by dkocher** [comment]
Please update to the latest snapshot build through _Preferences -> Update_ which should resolve this issue.
